### PR TITLE
Replace SVGMatrix with DOMMatrix

### PR DESCRIPTION
**Pull request by YonatanKra**
> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'

> What was wrong/why is this fix needed? (quick summary only)

> Anything else that could help us review it

### CODE REVIEW

File: files/en-us/web/api/path2d/addpath/index.html [hunk 1]
```diff
@@ -45,8 +45,7 @@ <h4 id="JavaScript">JavaScript</h4>
 
 <p>First, we create two separate {{domxref("Path2D")}} objects, each of which contains a
   rectangle made using the {{domxref("CanvasRenderingContext2D.rect()", "rect()")}}
-  method. We then create a matrix using {{Domxref("document.createElementNS()")}} and
-  <code>createSVGMatrix()</code>. We then add the second path to the first using
+  method. We then create a matrix using {{Domxref("DOMMatrix.DOMMatrix")}}. We then add the second path to the first using
```
Comment (teoli2003) [inline files/en-us/web/api/path2d/addpath/index.html:48]:
```suggestion
  method. We then create a matrix using {{Domxref("DOMMatrix.DOMMatrix", "DOMMatrix()")}}. We then add the second path to the first using
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
A tiny change is needed, but nice catch!

**Comment by YonatanKra** [comment]
> A tiny change is needed, but nice catch!

Done. Cheers!

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome!

**Pull request merged**